### PR TITLE
re-add oadm binary as alias for oc adm

### PR DESCRIPTION
**Pull request by mfojtik**
@deads2k this will provide 'oadm' command for some time with fat deprecation warning. Not sure if I have to plumb the hack/ scripts more or this should be enough.

### CODE REVIEW

**Comment by mfojtik** [comment]
/retest

@smarterclayton @deads2k quick review?

**Comment by deads2k** [comment]
Missing the bits in origin.spec.

@smarterclayton when do we get to remove this entirely?

File: cmd/oadm/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+	path, err := exec.LookPath(baseCommand)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+		os.Exit(1)
+	}
+
+	fmt.Fprintf(os.Stderr, "DEPRECATED: The 'oadm' command is deprecated, please use '%s adm' instead.\n", baseCommand)
+
+	if err := syscall.Exec(path, append([]string{"adm"}, os.Args[1:]...), os.Environ()); err != nil {
```
Comment (deads2k) [inline cmd/oadm/main.go]: @smarterclayton I remember you had opinions about which API to use to exec.

**Comment by mfojtik** [comment]
@deads2k rpmspec bits added.

**Comment by mfojtik** [comment]
@stevekuznetsov can you please check if there are other places I have to put this binary? I tried to follow @deads2k removal commit, but seems like I'm still missing something.

**Comment by stevekuznetsov** [comment]
@mfojtik can you help me help you by pointing out specific failure modes you're seeing?

**Comment by smarterclayton** [comment]
That is the correct call to Exec (equivalent to execve(2))

**Comment by mfojtik** [comment]
@stevekuznetsov well, it seems like the release build is broken but I can't see any failure in the log, except: `[ERROR] PID 13842: hack/lib/build/environment.sh:170: `return "${exitcode}"` exited with status 2`.

/retest

**Comment by mfojtik** [comment]
@stevekuznetsov it seems like I forgot to update something to re-add the `oadm` binary (guessing an rpm spec).

**Comment by mfojtik** [comment]
OK, seems like the rpm build now works and i'm getting flakes now. @deads2k @smarterclayton are you ok shipping this?

**Comment by wanghaoran1988** [comment]
@kargakis Seems the "/retest" command always run all the tests, does this designed for this? it's not the same with how upstream does.

**Comment by kargakis** [comment]
@wanghaoran1988 it reruns all failed tests, similar to how it's working upstream. We are using the same code.

**Pull request merged**